modules/apps/object/object-test/src/testFunctional/tests/ObjectsRootModel.testcase

```diff
@@ -0,0 +1,79 @@
+@component-name = "portal-object"
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Object";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+	}
+
+	tearDown {
+		var testLiferayVirtualInstance = PropsUtil.get("test.liferay.virtual.instance");
+
+		if (${testLiferayVirtualInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
```

Comment:
The tearDown does not have `ObjectAdmin.deleteAllCustomObjectsViaAPI();` because it is not possible to delete relationships and objects that are part of a hierarchical structure.
The virtual instance is successfully deleted.